ufo2ir/src/toir.rs

```diff
@@ -1,19 +1,109 @@
+use std::{collections::{HashSet, HashMap}, path::{Path, PathBuf}, fs};
+
 use ir::ir;
-use norad::designspace::{self, DesignSpaceDocument};
+use norad::{
+    designspace::{self, DesignSpaceDocument, Dimension},
+    fontinfo::NonNegativeIntegerOrFloat,
+};
 
 use crate::error::UfoToIrError;
 
-pub fn designspace_to_ir(designspace: DesignSpaceDocument) -> Result<Vec<ir::Axis>, UfoToIrError> {
-    // Truly we have done something amazing here today
-    let ir_axes: Vec<ir::Axis> = designspace.axes.into_iter().map(to_ir_axis).collect();
+struct Font {
```

Comment:
I think naming this internal struct `Font` could be confusing as elsewhere we use `fontir::Font` already. I think you can call it `Source` (or `FontSource`), the DS calls the corresponding elements `<source>`

Reply:
+1, will rename